src/plugins/event-plugins/ResourcePlugin.ts

```diff
@@ -132,6 +114,12 @@ export class ResourcePlugin extends InternalPlugin {
     };
 
     protected onload(): void {
-        window.addEventListener(LOAD, this.resourceEventListener);
+        // We need to set `buffered: true`, so the observer also records past
```

Comment:
Question: are will still concerned about using `performance.geEntries()` as backup for browsers that do not support `PerformanceObserver`?